07.label_semantic_roles/train.py

```diff
@@ -178,7 +181,7 @@ def event_handler(event):
                 print "Pass %d, Batch %d, Cost %f, %s" % (
                     event.pass_id, event.batch_id, event.cost, event.metrics)
             if event.batch_id % 1000 == 0:
-                result = trainer.test(reader=reader, feeding=feeding)
+                result = trainer.test(reader=test_reader, feeding=feeding)
```

Comment:
This is another bug but not related to issue.

`train_reader` and `test_reader` should be two instances because `test` will read all data in `reader`. If they are the same instance, training data will be reset every time.